config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -16,6 +17,13 @@ presubmits:
         # we need privileged mode in order to do docker in docker
         securityContext:
           privileged: true
+        resources:
+          limits:
+            cpu: "2"
```

Comment:
This is fairly low for compiling and will just make the job run longer.

Reply:
Updated.